src/Product/ProductList.js

```diff
@@ -0,0 +1,65 @@
+import { useState, useEffect } from "react";
+import Product from "./Product";
+
+const API_URL = "https://px32id5fdg.execute-api.us-east-1.amazonaws.com/data";
+
+function ProductList() {
+  const [productData, setProductData] = useState([]);
+  const [searchInput, setSearchInput] = useState("");
+  const [loading, setLoading] = useState(true);
+
+  useEffect(() => {
+    fetch(API_URL)
+      .then((response) => {
+        return response.json();
+      })
+      .then((data) => {
+        setProductData(data);
+        setLoading(false);
+      })
+      .catch((error) => {
+        console.log(error);
```

Comment:
Nice to include `.catch()`. If you had more time, how would you display that error to the user instead of in the console? What state, components, and logic would you need to add?